lib/composables/Table.ts

```diff
@@ -178,19 +185,19 @@ export interface TableCellState extends TableCellBase {
   mode?: ColorModes
 }
 
-export interface TableCellActions extends TableCellBase {
+export interface TableCellActions<R = any> extends TableCellBase {
   type: 'actions'
-  actions: TableCellAction[]
+  actions: TableCellAction<R>[]
 }
 
-export interface TableCellAction {
+export interface TableCellAction<R = any> {
   mode?: Mode
   icon?: any
   iconMode?: Mode
   label?: string
   labelMode?: Mode
-  onClick(record: any): void
-  show?(record: any): boolean
+  onClick(record: R): void
+  show?(record: R): boolean
```

Comment:
Hmmmm I vote for usability here. Also, I have a feeling we could remove `value` from everywhere in the future version.

Since we started using Ops pattern, I see my self barely using `value` anymore because ops requires whole model to be passed in 🤔 

```ts
{
  cell: (_, user) => ({
    value: userOps.fullName(user)
  })
}
```